scripts/configure-installer-tests-cluster.sh

```diff
@@ -15,19 +15,25 @@ export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig
 # List of users to create
 USERS="developer odonoprojectattemptscreateproject odosingleprojectattemptscreate odologinnoproject odologinsingleproject1"
 
-# Check if nessasary files exist
-if [ ! -f $KUBEADMIN_PASSWORD_FILE ]; then
-    echo "Could not find kubeadmin password file"
-    exit 1
-fi
+# Attempt resolution of kubeadmin, only if a CI is set
+if [ -z $CI ]; then
```

Comment:
Can you add some information why is this needed, and how it is used?

Reply:
SHort story apparently OpenShift CI has black magic to export kubeconfig and login as kubeadmin. Hence this needs to be skipable

Reply:
Done

Reply:
This is the information that we need put into the comment. No one will know what this is in a few weeks :-(